src/components/Main/Main.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const Main: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => {
+        const { title, completed, id } = todo;
+
+        return (
+          <div
```

Comment:
Better to use `<ul>` and `<li>` with map method
```suggestion
    <ul className="todoapp__main" data-cy="TodoList">
      {todos.map(todo => {
        const { title, completed, id } = todo;

        return (
          <li
```